src/plugins/intel_gna/src/gna_device.cpp

```diff
@@ -502,42 +474,42 @@ void GNADeviceHelper::dumpTLVForDeviceVersion(const uint32_t modelId,
                                               std::ostream& outStream,
                                               const std::vector<GnaEndpoint>& inputsContainer,
                                               const std::vector<GnaEndpoint>& outputsContainer) {
-    const auto compileTarget = GetCompileTarget();
     ExportTlvModel(modelId,
                    nGnaDeviceIndex,
                    outStream,
-                   compileTarget,
+                   target->get_effective_compile_target(),
                    inputsContainer,
                    outputsContainer,
                    allAllocations);
 }
 
-void GNADeviceHelper::createVirtualDevice(Gna2DeviceVersion devVersion) {
-    const auto status = Gna2DeviceCreateForExport(devVersion, &nGnaDeviceIndex);
-    GNADeviceHelper::checkGna2Status(status, "Gna2DeviceCreateForExport(" + std::to_string(devVersion) + ")");
+void GNADeviceHelper::createVirtualDevice(DeviceVersion devVersion) {
```

Comment:
it is not bug but inconsistence in some method above you passes DeviceVersion but const reference but here by value.